person.rb

```diff
@@ -0,0 +1,33 @@
+class Person
+    def initialize(name = "Unknown", age, parent_permission = true)
+        @id = Random.new.rand(1..100)
+        @name = name
+        @age = age
+    end
+
+    def parent_permission
+        @parent_permission
+    end
+
+    attr_reader :id, :name, :age
+    attr_writer :name, :age
+
+    private
+    def is_of_age?
+        if @age >= 18
+            true
+        else
+            false
+        end
+    end
+
+    def can_use_services?
+        if @age >= 18 && @parent_permission
+            true
+        else
+            false
+        end
+    end
```

Comment:
- [ ] Good job so far 👍 Kindly place this `can_use_services?` method above the private method since it is a public method.
- [ ] It would also be great if you can make use of the private method `is_of_age?` here that is:  
``` npm
   def can_use_services?
      is_of_age?() || @parent_permission
   end
```

Reply:
Hello @Kingstalux, Thank you for your review, but using the private method in the person file throws linter errors as it is duplicated.

Reply:
```
def can_use_services?
      is_of_age?() || @parent_permission
   end
```
I just used it before 
 ```
def can_use_services?
    if @age >= 18 || @parent_permission
      true
    else
      false
    end
  end
```